librz/asm/arch/z80/z80.c

```diff
@@ -68,64 +68,65 @@ static int z80OpLength (const ut8 *buf, int len) {
 }
 
 // #include'd in asm/p/asm_z80.c
-FUNC_ATTR_USED static int z80Disass (RzAsmOp *op, const ut8 *buf, int len) {
-	int ret = z80OpLength (buf, len);
+FUNC_ATTR_USED static int z80Disass(RzAsmOp *op, const ut8 *buf, int len) {
+	int ret = z80OpLength(buf, len);
 	const z80_opcode *z_op;
 	const char **cb_tab;
 	ut8 res;
 	if (!ret) {
 		return ret;
 	}
 	z_op = z80_op;
-	const char *buf_asm = "invalid";
 	switch (z_op[buf[0]].type) {
 	case Z80_OP8:
-		buf_asm = sdb_fmt ("%s", z_op[buf[0]].name);
+		rz_asm_op_setf_asm(op, "%s", z_op[buf[0]].name);
```

Comment:
Just set those directly, here and in other similar places, no need to add unnecessary computations.